subworkflows/MODEL_TO_DATA.nf

```diff
@@ -8,6 +8,8 @@ params.project_name = "DPE-testing"
 params.view_id = "syn53475818"
 // Synapse ID for Input Data folder
 params.input_id = "syn51390589"
+// E-mail template (0 to send e-mail without score update, 1 to send an e-mail with)
+params.email_with_score = 0
```

Comment:
This `email_with_score` input will be used by organizers to switch between two email templates:
`0`: E-mail template where score is not shared, nor any links to submission views
`1`: E-mail template where score IS shared, and link to the submission view

I'm using binary because it minimizes risk of typo errors that would otherwise be introduced by a string like 'yes'/'no' or 'true'/'false'.

Reply:
I see your rationale here, but I think a descriptive string that indicates if the score should be shared or not would make this more readable. Especially because you are passing this to a Python script. You can always check if a valid value is passed either in the Nextflow workflow or in the python script itself.

Reply:
Fair point, I've updated the parameter requirements to take in either "yes"/"no" and raise a `ValueError` when the input is not as expected.

The input is not case-sensitive:

<img width="1079" alt="image" src="https://github.com/Sage-Bionetworks-Workflows/nf-synapse-challenge/assets/32107699/ad08b907-c3bc-4ce7-9535-0c77a452ed5a">

But the logic will catch typos when present:

<img width="690" alt="image" src="https://github.com/Sage-Bionetworks-Workflows/nf-synapse-challenge/assets/32107699/201d397c-6459-4837-a66c-937549ebd8dc">

Reply:
Since this is a matter of having a correct configuration for the workflow run, you may also want to check that `params.email_with_score` is either yes or no in the workflow. [Here](https://github.com/mc2-center/nf-histoqc/blob/3e54e9abd14a188bb8b947e0d49478b1f1a48f95/main.nf#L11) is an example you could follow for throwing an error if that is not the case

Reply:
I know its not necessarily case-sensitive now, but we might want to make sure people are passing a consistent parameter `yes` and `no` just in case it matters in the future. I think you should be able to enforce that in the check within the Nextflow workflow. Also, make sure you update any documentation with information about the new parameter

Reply:
I think for input parameters that aren't calls to specific names (like `project_name` for example), it wouldn't be necessary to make it case-sensitive as the param is used as a trigger to switch on some other logic, and so it can be manipulated (e.g. I set the string to lowercase with `.lower()`). I think that just provides more flexibility/ease-of-use from a users' standpoint.

However, I'm not terribly invested in the idea of making this param case-sensitive, so if you strongly think we should then I can remove that flexibility.

Reply:
I am thinking about it more like an `Enum` in Python where the string "yes" is linked to a specific meaning. I think that flexibility/manipulation is a potentially confusing factor, especially because we have two separate software components of the NF workflow and Python script. If that string is always consistent in both places I think that simplicity would be preferable. 

I agree that it's not strictly necessary, however.

Reply:
I see - in that case yes, I can remove the insensitivity and specify which parameters are case-sensitive in the docs